test/Index/index_static_funcs.swift

```diff
@@ -0,0 +1,130 @@
+// RUN: %empty-directory(%t)
+//
+// RUN: %target-swift-ide-test -print-indexed-symbols -source-filename %s | %FileCheck %s
+
+protocol SwiftProto {
+// CHECK: [[@LINE-1]]:10 | protocol/Swift | SwiftProto | [[Proto_USR:.*]] | Def | rel: 0
+  static func staticMethod()
+  // CHECK: [[@LINE-1]]:15 | static-method/Swift | staticMethod() | [[ProtoStaticMethod_USR:.*]] | Def,RelChild | rel: 1
+  static func classMethod()
```

Comment:
You have 2 `static` functions here, is this useful?

Reply:
One is intended to be `static` in the implementation, the other `class`.In the protocol definition they both have to be declared as `static`. The rest of the test checks that calls to `staticMethod` don't have `Dynamic` added, but `classMethod` does (when not called from the type specifically).

Reply:
It is only useful in the class case though (not protocol, struct, or enum), so you're right - I could just have it in the class instead.